nekoyume/Assets/_Scripts/Planet/PlanetSelector.cs

```diff
@@ -161,25 +161,39 @@ private static PlanetContext UpdateCommandLineOptions(PlanetContext context)
                 return context;
             }
             
-            context.CommandLineOptions.genesisBlockPath = currentPlanetInfo.GenesisUri;
-
-            if (currentPlanetInfo.RPCEndpoints.HeadlessGrpc.Count == 0)
+            var clo = context.CommandLineOptions;
+            clo.genesisBlockPath = currentPlanetInfo.GenesisUri;
+            var rpcEndpoints = currentPlanetInfo.RPCEndpoints;
+            if (rpcEndpoints.HeadlessGrpc.Count == 0)
             {
                 context.Error = "RPCEndpoints.HeadlessGrpc is empty." +
                                 "Check the planet registry url in command line options: " +
-                                context.CommandLineOptions.PlanetRegistryUrl;
+                                clo.PlanetRegistryUrl;
                 return context;
             }
 
-            var uris = currentPlanetInfo.RPCEndpoints.HeadlessGrpc
+            var uris = rpcEndpoints.HeadlessGrpc
                 .Select(url => new Uri(url))
                 .ToArray();
-            context.CommandLineOptions.RpcServerHosts = uris.Select(uri => uri.Host);
+            clo.RpcServerHosts = uris.Select(uri => uri.Host);
 
             // FIXME: It determines the RPC server host randomly for now.
             var uri = uris[Random.Range(0, uris.Length)];
-            context.CommandLineOptions.RpcServerHost = uri.Host;
-            context.CommandLineOptions.RpcServerPort = uri.Port;
+            clo.RpcServerHost = uri.Host;
+            clo.RpcServerPort = uri.Port;
+            clo.ApiServerHost = rpcEndpoints.HeadlessGql.Count > 0
```

Comment:
Below selections don't seem to be randomly (despite of the above mention) maybe we need to shuffle or write additional comment for them.

Reply:
I applied it.